src/CommonDBChild.php

```diff
@@ -53,14 +53,22 @@ abstract class CommonDBChild extends CommonDBConnexity
     /** @var string $items_id */
     public static $items_id; // Field name
     // * rights
+    /** @var int */
```

Comment:
This could be even more precise.

```suggestion
    /** @var CommonDBConnexity::DONT_CHECK_ITEM_RIGHTS|CommonDBConnexity::HAVE_VIEW_RIGHT_ON_ITEM|CommonDBConnexity::HAVE_SAME_RIGHT_ON_ITEM */
```